templates/navigation-bar-new.twig

```diff
@@ -11,7 +12,7 @@
 		data-ga-label="{{ page_category }}"
 		aria-expanded="false"
 		data-bs-toggle="open"
-		data-bs-target="#nav-main">
+		data-bs-target="#burger-menu">
```

Comment:
Is it possible to preserve the `#nav-main` ID for this?

The current version is deployed on dev sites, and [on GPNL](https://www-dev.greenpeace.org/nl/) it's not possible anymore to open the menu on mobile. Didn't confirm yet but probably related to this change.

Reply:
Also an unstyled button is visible on all screen sizes
![Screenshot from 2022-04-04 14-49-03](https://user-images.githubusercontent.com/7604138/161547254-ed28e30f-c2b9-4372-bde5-c89b8ca8e18d.png)

Reply:
But if you change to `#nav-main` you will have 2 elements with the same ID and that is not allowed by the `DOM`.

About the problem of the GPNL, as a simple sight the issue is because we've removed the styles from [here](https://github.com/greenpeace/planet4-master-theme/pull/1628/files#diff-91009267fa7864d1b148d657a615793b252f3e148c4001b639eab5a4c51324e4L74-L324) and put it [here](https://github.com/greenpeace/planet4-master-theme/pull/1628/files#diff-30638aecdc1bb63b61eab64177e65ff6123f16d8f032495ff5ef8a97b66272b8R1-R268), because we rethought the logic of the Mobile navbar.

Reply:
Also they've overridden the [navigation bar template](https://github.com/greenpeace/planet4-child-theme-netherlands/blob/develop/templates/navigation-bar-new.twig) and its [styles](https://github.com/greenpeace/planet4-child-theme-netherlands/blob/develop/assets/child-theme-main/styles/components/_navbar.scss).

Reply:
One more thing, `document.querySelector('#burger-menu')` is returning `null` and this is because they're not adding logic of adding the burger menu template ([here](https://github.com/greenpeace/planet4-child-theme-netherlands/blob/develop/templates/navigation-bar-new.twig#L161-L165)) when `Enable dropdown menus` is enabled.

Reply:
We ended up needing a fix for child themes anyway, as they were overriding the navigation bar template entirely.